include/bluetooth/conn.h

```diff
@@ -413,8 +446,10 @@ struct bt_conn_cb {
 	 *
 	 *  @param conn Connection object.
 	 *  @param level New security level of the connection.
+	 *  @param reason Security error. Zero for success, non-zero otherwise.
 	 */
-	void (*security_changed)(struct bt_conn *conn, bt_security_t level);
+	void (*security_changed)(struct bt_conn *conn, bt_security_t level,
+				 u8_t err);
```

Comment:
This should be called `status` and not `err`

Reply:
Same comment as https://github.com/zephyrproject-rtos/zephyr/pull/17306#discussion_r317117617 I'd prefer to keep using `err`